src/client/src/spu/spu_pool.rs

```diff
@@ -62,7 +62,7 @@ impl SpuPool {
     /// start synchronize based on pool
     pub async fn start(
         config: ClientConfig,
-        sc_socket: &AllMultiplexerSocket,
+        sc_socket: SharedAllMultiplexerSocket,
```

Comment:
I believe this set of changes was only helping because it just happened to keep the Arc with the AllMultiplexerSocket alive. Since it is now held in FluvioInner, this should not be necessary

Reply:
Yeah, I put this back to what it was.